### ISSUE TITLE
test

### ISSUE THREAD

**Issue by Micro0x00** [koalaman/shellcheck#2632]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2148, SC2162, SC2164, SC2164
- My shellcheck version (`shellcheck --version` or 'online'): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

read -p "Do You want install Gitleaks (Y/n) " choice
     case $choice in
     no | No | NO | n)
     echo -e "skip"
     ;;
     yes| YES | Yes | y | Y)
    git clone https://github.com/zricethezav/gitleaks.git
    cd gitleaks
    make build
    mv gitleaks /usr/local/bin
    cd -
     ;;
     esac

```

#### Here's what shellcheck currently says:
```
Line 1	SC2148: Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.
Line 2	SC2162: read without -r will mangle backslashes.
Line 9	SC2164: Use 'cd ... || exit' or 'cd ... || return' in case cd fails.
Line 12	SC2164: Use 'cd ... || exit' or 'cd ... || return' in case cd fails.
```
#### Here's what I wanted or expected to see:

TODO: Describe expected/desired output

**Comment by brother** [comment]
Follow the instructions and the code will at least be Better[tm].
